### PR TITLE
tests/fs: NFFS has been replaced with Littlefs in multifs tests

### PR DESCRIPTION
**Pull request by de-nordic**
With removal of NFFS from Zaphyr it would be no longer possible to
test multifs with NFFS and LittleFS will be now used instead.

Supplement to https://github.com/zephyrproject-rtos/zephyr/pull/21793

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

### CODE REVIEW

File: tests/subsys/fs/multi-fs/src/test_utils.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#include <fs/fs.h>
+#include <ztest.h>
+#include <storage/flash_map.h>
+
+int check_file_dir_exists(const char *fpath)
+{
+	int res;
+	struct fs_dirent entry;
+
+	/* Verify fs_stat() */
```
Comment (nvlsianpu) [inline tests/subsys/fs/multi-fs/src/test_utils.c]: this comment doesn't explain anything at all

File: tests/subsys/fs/multi-fs/testcase.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-    platform_whitelist: qemu_x86
-    tags: nffs fatfs filesystem
+    platform_whitelist: native_posix qemu_x86
+    tags: littlefs fatfs filesystem qemu_x86
   filesystem.fs_shell:
-     extra_args: CONF_FILE="prj_fs_shell.conf"
-     platform_whitelist: qemu_x86
-     tags: nffs fatfs filesystem
+    extra_args: CONF_FILE="prj_fs_shell.conf"
+    platform_whitelist: native_posix qeny_x86
```
Comment (nvlsianpu) [inline tests/subsys/fs/multi-fs/testcase.yaml]: perhaps qemu_x86?

**Review by nvlsianpu — CHANGES_REQUESTED** [review]

**Review by nvlsianpu — APPROVED** [review]

**Comment by nvlsianpu** [comment]
@carlescufi this is supplemet to [NFFS removel PR](21793)

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
@de-nordic please rebase

**Comment by nvlsianpu** [comment]
@carlescufi rebased

**Pull request merged**